src/main/resources/translator.properties

```diff
@@ -248,6 +248,12 @@ optional = optional
 doNotFingerprintArtifacts = fingerprintArtifacts
 resultVariableSuffix = resultVariableSuffix
 
+# spec options require a custom type DSLEnclosedObjectStrategy -- see the class for more info
+spec = spec
+spec.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLEnclosedObjectStrategy
+spec.enclosing_tag = cron
+specEnclosed = spec
+
```

Comment:
If I had to guess what cron -> spec actually refers to, I would say it's this:

```
<triggers>
<hudson.triggers.TimerTrigger>
<spec>H H(0-7) * * *</spec>
</hudson.triggers.TimerTrigger>
</triggers>
```

see this job: https://jenkins-tinyspeck-bedrock-dev.tinyspeck.com/job/pam-test/config.xml

<img width="1309" alt="Screen Shot 2022-11-03 at 1 21 59 PM" src="https://user-images.githubusercontent.com/63604061/199826198-90dbb567-30a9-4a2b-987f-0150f0b9cfe6.png">

Therefore we can assume that cron->spec is actually not supported for reverse trigger and this is not needed

Reply:
I see. I've removed those and kept just the support for the other tags within reverseBuildTrigger.

Reply:
Because spec is not supported for reverse trigger, I have added `spec` back to propertiesToBeSkipped and added the properties under reverseBuildTrigger to match the apiViewer in the format of 
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/112515811/201417904-088d213d-3ac7-4c42-9fad-bd84b82db544.png">

I tested the xml you provided for hudson.triggers.TimerTrigger and spec is being recognized and converted correctly to : 
```
}
	triggers {
		cron("H H(0-7) * * *")
	}
```